src/main.rs

```diff
@@ -622,6 +628,14 @@ fn main() {
         backface_culling: glium::draw_parameters::BackfaceCullingMode::CullCounterClockwise,
         ..Default::default()
     };
+    let arrowhead_draw_params = glium::DrawParameters {
```

Comment:
The arrows are currently occluded by the geometry, which isn't the end of the world, but like a lot of 'helpers' it might be better to show this through geometry. Either by changing its depth checking to always overwrite, or by doing what lollipops do and do a darker, second pass with reverse depth checking if you're feeling up to it (the second pass starts at main.cpp:1115 to compare)